crates/aranya-daemon/src/policy.md

```diff
@@ -242,6 +257,53 @@ function open_envelope(sealed_envelope struct Envelope) bytes {
     return verified_command
 }
 ```
+#### Role Operation RBAC Functions
+
+Used to determine if a role has permission to publish a certain command.
+A command may only have 1 role with permission to publish it.
+A role may be assigned to multiple devices.
+A device may be assigned multiple roles.
+
+```policy
+finish function create_role_fact(role struct RoleInfo) {
+    create Role[role_id: role.role_id]=>{name: role.name, author_id: role.author_id}
+}
+
+finish function delete_role_fact(role struct Role) {
+    delete Role[role_id: role.role_id]
+}
+
+finish function assign_op_role(op string, role_id id) {
+    create OpRequiresRole[op: op]=>{role_id: role_id}
+}
+
+finish function revoke_op_role(op string) {
```

Comment:
This is only used in one place, just inline it there. (Plus, the naming isn't clear that it's deleting the `OpRequiresRole` fact.)